main.go

```diff
@@ -239,11 +239,13 @@ func main() {
 				Name: "extract-circuit",
 				Flags: []cli.Flag{
 					&cli.StringFlag{Name: "output", Usage: "Output file", Required: true},
+					&cli.StringFlag{Name: "proof-size", Usage: "Length of Proof vector (i.e. depth of tree)", Required: true},
```

Comment:
This should follow the naming convention of the rest of this and be called `tree-depth`